frontend/src/components/MapView/Legends/index.tsx

```diff
@@ -94,6 +98,22 @@ function Legends({ classes, layers, extent }: LegendsProps) {
       'application/json',
     );
   };
+  const handleAnalysisDownloadCsv = (): void => {
```

Comment:
Could simplify this by extracting the definition of an analysisDate? Since all the other params overlap?

Reply:
Done :)